manager.py

```diff
@@ -283,6 +283,8 @@ def run(self):
                 if self.quit_after_interval:
                     sys.exit(0)
                 this_start, this_end = self.get_interval(this_end)
+                if time.altzone < self.interval/60:
+                    self.quit_after_interval
```

Comment:
This code only executes at the end of an interval anyway... so I'm not sure what the point is.